torch/_dynamo/symbolic_convert.py

```diff
@@ -341,6 +342,14 @@ def inner(self: "InstructionTranslatorBase", inst: Instruction):
                 self.jump(inst)
                 return
 
+            if isinstance(value, SymNodeVariable):
+                # if the assertion is normal shape expression.
+                # just install guard and bail out.
+                value.evaluate_expr()
```

Comment:
uhh you need to actually fail if it evaluates to False lol

Reply:
ok